src/main/kotlin/com/avail/interpreter/primitive/pojos/P_InvokeCallback.kt

```diff
@@ -74,13 +75,22 @@ object P_InvokeCallback : Primitive(-1, Private, CanSuspend)
 			runtime.callbackSystem().executeCallbackTask(
 				callbackPojo.javaObjectNotNull(),
 				argumentsTuple,
-				CallbackCompletion { succeed(it.makeShared()) },
-				CallbackFailure { throwable ->
-					fail(
-						newPojo(
-							identityPojo(throwable),
-							pojoTypeForClass(throwable.javaClass)
-						).makeShared())
+				object: CallbackCompletion {
```

Comment:
This is a strange expansion.  Was it broken in the previous .kt version?

Reply:
No, SAML doesn't work here :(
I had to do research and this was the answer. Kinda sucks...

Reply:
https://dev.to/ranilch/functional-interfaces-self-loathing-in-kotlin-4bce